main/src/ca/uwaterloo/flix/language/errors/WeederError.scala

```diff
@@ -1028,4 +1028,25 @@ object WeederError {
          |""".stripMargin
     }
   }
+
+
+  /**
+    * An error raised to indicate expression nesting beyond the threshold.
+    *
+    * @param hint     Optional hint with more details about the error
```

Comment:
What do you mean additional error message? I copied from other errors that do the same thing

Reply:
We should have an error message for each specific type of error. Otherwise we cannot build proper tooling.

For example, if we wanted a quickfix I would now have to rely on matching on the string.

Reply:
We can add it when that's the case?

keep in mind this hint is also just a hint - we don't know for sure whether there is Nil in the end or something else